README.Rmd

```diff
@@ -13,1044 +12,68 @@ knitr::opts_chunk$set(
-- years until litigation event: 5
-- settlement factor (to vary scenarios of share of damages paid out): 1
-- growth rate of annual dividends: 0.02
-- discount rate: 0.03
-- percentage of the NPV in the terminal value: 0.1
- 
-We use the EBIT of 2019 of the company as the starting point to apply the DCF.
-First we get the future net cash flows by applying the growth rate. We then
-subtract the settlement amount calculated in the first part of the exercise
-from the EBIT in the year of the litigation event.
-
-**NOTE:** we are currently applying the years_until_litigation factor to HER
-calculations as well. This may have to be changed, if it is decided these
-liabilities should not grow anymore.
-
-After the litigation event, the net dividends of the company continue to grow
-at the indicated annual growth rate. At the moment, the level of dividends at
-that point in time is not reset, which implies the company continues to build up
-litigation risk after the settlement. If it is a requirement in the future to
-reflect that companies follow their targets after the litigation, this mechanism
-still has to be implemented.
-
-We then discount the dividends of each year (once with litigation, once without),
-to obtain the net present values for a baseline scenario without litigation and
-for the litigation scenario as defined by the parameters.
-
-These NPVs are used to compute the percentage value change between the
-scenarios. Such value changes can, in certain cases, be more than 100%, which would
-translate into the bankruptcy of the company. The model may not reflect all
-mechanisms at play in determining such an outcome.
-
-
-#### Steps in calculating impact on portfolios of shareholders
-
-Ultimately, a large part of the cost to the companies produced by such a
-settlement will be carried by the shareholders of the company. For that reason,
-we need to be able to translate the equity impact from the share price level
-to the portfolio level.
-We do this in a very straightforward manner, using information generated from
-a PACTA analysis of a portfolio. This analysis provides us with a variable that
-gives us the value share of the overall portfolio for each company-technology
-combination. If we aggregate this to the company level, we can deduce the share
-of the portfolio value that this asset accounts for and thus we know how large
-the exposure to the asset at risk is within the portfolio (company_exposure =
-plan_carsten * portfolio_aum).
-
-The expected loss on the portfolio level is then calculated as:
-
-expected loss = share price change * company exposure
+```
 
-By default, the module will estimate that impact for all relevant
-companies in the portfolio at hand.
+[Get started] ## tbc: add link to vignette
 
-----
 
-[CONTRIBUTING](https://github.com/2DegreesInvesting/r2dii.climate.stress.test/blob/master/.github/CONTRIBUTING.md)
+## Funding
+This project has received funding from the [European Union LIFE
```

Comment:
Are we sure about all the funding sources? LIFE is definitely true, but the other ones? Maybe check with Andy

Reply:
is it ok if I simply remove funding from the readme actually?

Reply:
For today it's fine, we are researching who should be on here and may have to add this again in the next few days

Reply:
ok, will add it once I have feedback from andy